flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/dataformat/TypeGetterSetters.java

```diff
@@ -36,6 +38,10 @@
  */
 public interface TypeGetterSetters {
 
+	int MAX_FIX_PART_DATA_SIZE = 7;
+	long HIGHEST_FIRST_BIT = Long.MIN_VALUE;
+	long HIGHEST_SECOND_TO_EIGHTH_BIT = 0x7FL << 56;
```

Comment:
I don't quite get the meaning of these variables, can you add some comments for it? What does it for, and what does it looks like (like in a form of 10000000 00000000 .....)